ui/app/components/job-status/panel/deploying.js

```diff
@@ -107,8 +135,10 @@ export default class JobStatusPanelDeployingComponent extends Component {
         status === 'running'
           ? alloc.isHealthy
             ? 'healthy'
-            : 'unhealthy'
-          : 'unhealthy';
+            : alloc.isUnhealthy
+            ? 'unhealthy'
+            : 'health_unknown'
+          : 'health_unknown';
```

Comment:
I'm not thrilled that I'm nesting a ternary here but it is only causing me mild discomfort